tests/test_spstats.py

```diff
@@ -171,3 +194,30 @@ def test_patches_method(self):
             gsd=diff.res[0],
             area_size=10000
         )
+
+class TestBinning:
+
+    def test_nd_binning(self):
+
+        ref, diff, mask = load_ref_and_diff()
+
+        slp, asp = xdem.coreg.calculate_slope_and_aspect(ref.data.squeeze())
```

Comment:
I think now we should try to have variable names that are explicit rather than short. So using 'slope' and 'aspect' might be clearer for reading.

Reply:
done